### ISSUE TITLE
display images in sublime via iTerm2 imgcat

### ISSUE THREAD

**Issue by randy3k** [Wramberg/TerminalView#24]
iTerm2 supports a way to display an image in the terminal via a the [imgcat](https://www.iterm2.com/documentation-images.html) tool. Basically, it sends the following sequence to the terminal.

```
ESC]1337 ; File = [optional arguments] : base-64 encoded file contents ^G
```

Currently, it shows the raw data on the screen.
<img width="1071" alt="screen shot 2017-07-12 at 2 33 57 am" src="https://user-images.githubusercontent.com/1690993/28104756-9e994ac4-66aa-11e7-8f1d-e8bcb0e21638.png">

What we only need is to display the content using Phantom.
```

view.add_phantom("TerminalView", sublime.Region(view.size(),view.size()), """<img src="data:image/png;base64,{}" />""".format(data), sublime.LAYOUT_BLOCK)
view.erase_phantoms("TerminalView")
```

**Comment by randy3k** [comment]
we should be able to add a handler [here](https://github.com/Wramberg/TerminalView/blob/d2a9e4176afc9a09741900fa8bd9a0587332a738/sublime_terminal_buffer.py#L69) or [in pyte](https://github.com/selectel/pyte/blob/26866c70e37b7ba00aaea3cf89fed50cd4f5cd7f/pyte/streams.py#L201) to capture the escape sequence

**Comment by rwols** [comment]
I agree this should be handled by Pyte. Maybe open an issue/PR there as well and link to here? @superbobry

**Comment by randy3k** [comment]
This is not an pyte issue per se. This iTerm2 protocol is by no mean standard, however, pyte should support specifying handlers for custom escaped sequences. For our usage, we just need to subclass and extend the relevant methods.

**Comment by rwols** [comment]
> pyte should support specifying handlers for custom escaped sequences.

Yeah, that's what I meant 👍
